corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,4 +1,4 @@
-DOMAIN_INDEX = "hqdomains_391fc13ad99b11e48da43c970e7be43b"
+DOMAIN_INDEX = "hqdomains_20150403_1448"
```

Comment:
we've been phasing out the dynamic hash generation off the mapping, if that's what you're asking